contracts/SaveToken.sol

```diff
@@ -294,9 +303,36 @@ contract SaveToken is ERC20Extended, Pausable {
         ); 
     }
 
-    function _subtractFromBalances(address user, uint256 amount) internal {
-        StorageLib.updateAssetBalance(user, StorageLib.getAssetBalance(user).sub(amount));
-        StorageLib.updateInsuranceBalance(user, StorageLib.getInsuranceBalance(user).sub(amount)); 
+    function _subtractFromBalances(address user, uint256 assetAmount, uint256 insuranceAmount) 
```

Comment:
I have this code directly in the mint function: 
```        // update asset and insurance token balances
        StorageLib.updateAssetBalance(
            msg.sender, StorageLib.getAssetBalance(msg.sender).add(assetTokens)
        );
        StorageLib.updateInsuranceBalance(
            msg.sender, StorageLib.getInsuranceBalance(msg.sender).add(insuranceTokens)
        ); 
```
For consistency, we should either remove the `_subtract` internal function and all the code directly in the `withdrawForUnderlyingAsset` function since it isn't being used anywhere else. Or, we should make an `_add..` internal function. Thoughts?

Reply:
Yeah, good point. Given that that internal function is only be used once, I'll add the `updateAssetBalance` and `updateInsuranceBalance` stuff back into the function like you have for `mint`.